### PR TITLE
ES-829615: Convert Wordto PDF in Azure Function

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
Added the Subpage for WordtoPDF in Azure Function v1 and v4.

Regards
venkatesh

### CODE REVIEW

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+title: Convert Word document to PDF in Azure Functions v1 | Syncfusion
+description: Convert Word document to PDF in Azure Functions v1 using .NET Word (DocIO) library  without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word to PDF in Azure Functions v1
+
+Syncfusion  DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can perform **convert a Word document to PDF in Azure functions v1**
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md:11]: Updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+## Steps to convert a Word document to PDF in Azure functions v1
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md:13]: Updated

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+title: Convert Word document to PDF in Azure Functions v4 | Syncfusion
+description: Convert Word document to PDF in Azure Functions v4 using .NET Word (DocIO) library  without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word document to PDF in Azure Functions v4
+
+Syncfusion DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read,  edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can perform **convert a Word document to PDF in Azure Functions v4**
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md:11]: Updated

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
+---
+title: Convert Word document to PDF in Azure Functions v4 | Syncfusion
+description: Convert Word document to PDF in Azure Functions v4 using .NET Word (DocIO) library  without Microsoft Word or interop dependencies.
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md:3]: Updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+Step 1: Create a console application to request the Azure functions API.
+
+Step 2: Add the following code snippet into **Main** method to post the request to Azure functions with template Word document and get the resultant PDF document.
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+//Reads the template Word document.
+FileStream fs = new FileStream(@"../../Data/DoctoPDF.docx", FileMode.Open, FileAccess.ReadWrite, FileShare.ReadWrite);
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md:118]: Updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+    throw;
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-PDF-Conversion/Convert-Word-document-to-PDF/Azure/Azure_Functions/Azure_Functions_v1).
+
+
+By executing the program, you will get the **PDF document** as follows.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md:156]: Remove these screenshot sentence

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+}
+catch (Exception ex)
+{
+    throw;
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-PDF-Conversion/Convert-Word-document-to-PDF/Azure/Azure_Functions/Azure_Functions_v1).
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md:153]: From GitHub, you can download the [console application](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-PDF-Conversion/Convert-Word-document-to-PDF/Azure/Azure_Functions/Console_Application) and [Azure Functions v1](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-PDF-Conversion/Convert-Word-document-to-PDF/Azure/Azure_Functions/Azure_Functions_v1).

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+title: Convert Word document to PDF in Azure Functions v4 | Syncfusion
+description: Convert Word document to PDF in Azure Functions v4 using .NET Core Word (DocIO) library without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word document to PDF in Azure Functions v4
+
+Syncfusion DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read,  edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can perform **convert a Word document to PDF in Azure Functions v4**
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md:11]: Using this library, you can **convert a Word document to PDF in Azure Functions v4**

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+## Steps to convert a Word document to PDF in Azure Functions v4:
+
+Step 1: Create a new Azure Functions project.
+![Create a Azure functions project](Azure_Images/Functions_v1/Azure_Function_WordtoPDF.png)
+
+Step 2: Create a project name and select the location.
+![Create a project name](Azure_Images/Functions_v1/Configuration_WordtoPDF.png)
+
+Step 3: Select function worker as **.NET 6.0(Long-term support)** 
+![Select function worker](Azure_Images/Functions_v2/Additional-Information-WordtoPDF.png)
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md:22]: Check all images having proper syntax

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+![Install Syncfusion.DocIORenderer.Net.Core NuGet package](Azure_Images/Functions_v2/Nuget-Package-WordtoPDF.png)
+
+Step 5: Include the following namespaces in the **Function1.cs** file.
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+using Syncfusion.Pdf;
+using Syncfusion.DocIO.DLS;
+using Syncfusion.DocIORenderer;
+using Syncfusion.DocIO;
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md:34]: using Syncfusion.Pdf;

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+title: Convert Word document to PDF in Azure Functions v1 | Syncfusion
+description: Convert Word document to PDF in Azure Functions v1 using .NET Word (DocIO) library  without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word to PDF in Azure Functions v1
+
+Syncfusion  DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can perform **convert a Word document to PDF in Azure Functions v1**
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md:11]: Updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md [hunk 12]
```diff
@@ -0,0 +1,160 @@
...
+
+
+Step 8: Select the **Create new** button
+![Configure Hosting Plan](Azure_Images/Functions_v1/Function_Instance_WordtoPDF.png)
+
+
+Step 9: Click **Create** button 
+![Select the plan type](Azure_Images/Functions_v1/Subscription_detail_WordtoPDF.png)
+
+Step 10:After creating app service then click **Finish** button
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md:96]: Updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md [hunk 13]
```diff
@@ -0,0 +1,160 @@
...
+![Creating app service](Azure_Images/Functions_v1/App_service_Created_WordtoPDF.png)
+
+
+Step 11: Click the **Publish** button
+![Click Publish Button](Azure_Images/Functions_v1/Before_Publish_WordtoPDF.png)
+
+Step 12: Publish has been succeed.
+![Publish succeeded](Azure_Images/Functions_v1/After_Publish_WordtoPDF.png)
+
+Step 13: Now, go to Azure portal and select the App Services. After running the service, click **Get function URL by copying it**. Then, paste it in the below client sample (which will request the Azure Functions, to perform **Word to PDF conversion** using the template Word document). You will get the output PDF document as follows.
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md:106]: Updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md (hunk 6 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md:156]: Updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md (hunk 7 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-Conversion-in-Azure-Functions-v1.md:153]: Updated

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md (hunk 8 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md:11]: Updated

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md (hunk 10 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md:34]: updated

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md [hunk 14]
```diff
@@ -0,0 +1,157 @@
...
+}
+catch (Exception ex)
+{
+    throw;
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+From GitHub, you can download the [console application](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-PDF-Conversion/Convert-Word-document-to-PDF/Azure/Azure_Functions/Console_Application) and [Azure Functions v1](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-PDF-Conversion/Convert-Word-document-to-PDF/Azure/Azure_Functions/Azure_Functions_v4).
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-to-PDF-in-Azure-function-v4.md:156]: Can you check and change the sentence properly as per this page.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

File: File-Formats/DocIO/WordToPDF_images/Output-Image-Azure-Fuctions.png
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/WordToPDF_images/Output-Image-Azure-Fuctions.png:1]: We are going to use same image for most of the cloud platforms. Can you rename the image as WordToPDF_Output_Cloud

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/WordToPDF_images/Output-Image-Azure-Fuctions.png
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/WordToPDF_images/Output-Image-Azure-Fuctions.png:1]: Updated

**Pull request merged**